src/ant/src/AntennaChecker.cc

```diff
@@ -218,10 +220,22 @@ void AntennaChecker::initAntennaRules()
                                   plus_diff_factor,
                                   diff_metal_reduce_factor};
     layer_info_[tech_layer] = layer_antenna;
-  }
 
-  block_ = db_->getChip()->getBlock();
-  dbu_per_micron_ = block_->getDbUnitsPerMicron();
+    double wire_width = dbuToMicrons(tech_layer->getWidth());
+    uint wire_thickness_dbu = 0;
+    tech_layer->getThickness(wire_thickness_dbu);
+    double wire_thickness = dbuToMicrons(wire_thickness_dbu);
+
+    if(tech_layer->getType() == dbTechLayerType::ROUTING && wire_thickness == 0)
+    {
+      logger_->error(ANT, 13, "No THICKNESS is provided for {}.", tech_layer->getConstName());
+    }
+
+    if(tech_layer->getType() == dbTechLayerType::ROUTING && wire_width == 0)
```

Comment:
Do we need to check width?  Any real wire will have a width determined by its geometry.